src/main/menus/app.js

```diff
@@ -136,6 +136,13 @@ var createTemplate = function(mainWindow, config) {
       click: () => {
         mainWindow.webContents.send('zoom-in', -1);
       }
+    }, {
+      label: 'Zoom Out (hidden)',
+      accelerator: 'CmdOrCtrl+_',
```

Comment:
Is `CmdOrCtrl+Shift+-` applicable for here? The accelerator assumes only US keyboard. For example, in Japanese keboard, need to press `Shift+\` to input "_".